frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -438,6 +465,64 @@ export const funnelLogic = kea<funnelLogicType>({
             },
         ],
         stepsWithCount: [() => [selectors.steps], (steps) => steps.filter((step) => typeof step.count === 'number')],
+        stepsWithConversionMetrics: [
+            () => [selectors.steps],
+            (steps): FunnelStepWithConversionMetrics[] => {
+                return steps.map((step, i) => {
+                    const previousCount = i > 0 ? steps[i - 1].count : 0
```

Comment:
I believe these calculations could help in the main funnel display (FunnelBarGraph.tsx) too to avoid calculating the same thing multiple times